daterangepicker.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @version: 2.1.31
+ * @version: 1.0.2
```

Comment:
because when you run the `npm version patch` command it will only update the package.json file.
I think we have to update the version information manually before running `npm publish` and `git push`.